aspnet/fundamentals/localization.rst

```diff
@@ -0,0 +1,337 @@
+ASP.NET Localization
+====================
+
+By `Tom Archer`_
+
+The ASP.NET 5 localization support provides middleware for establishing the correct culture and UI culture on the thread based on the request, and mechanisms for accessing localized content based on the current culture. In this article, you'll learn how to enable localization and establish the culture using both built-in culture providers. You'll also see how to implement your own custom culture provider.
+
+Enabling localization and establishing culture
+-----------------------------------------------
+
+To enable localization, add the request localization middleware to the request pipeline from the ``Startup`` class with a call to the ``IApplicationBuilder.UseRequestLocalization`` extension method. The ``IApplicationBuilder.UseRequestLocalization`` extension method has two overloaded versions: a parameter-less version that instantiates and uses a default ``RequestLocalizationOptions`` object and a version that allows you to specify a ``RequestLocalizationOptions`` object as a parameter.
+
+.. code-block:: c#
+	:emphasize-lines: 5-6
+
+	public void Configure(IApplicationBuilder app, IStringLocalizer<Startup> SR)
+	{
+		...
+
+		var options = new RequestLocalizationOptions();
+		app.UseRequestLocalization(options);
+
+		...
+	}
+
+The ``RequestLocalizationOptions`` class defines a collection property - ``RequestLocalizationOptions.RequestCultureProviders`` - that represents an ordered list of objects that implement the ``IRequestCultureProvider`` interface.
+
+There are three intrinsic implementations of the ``IRequestCultureProvider`` interface - each of which attempts to establish the correct culture and UI culture on the thread based on various aspects of the incoming request:
```

Comment:
I would move part of this sentence a little earlier: 

". . . that represents an ordered list of objects that implement the IRequestCultureProvider interface. Each request culture provider attempts to provide the correct culture and UI culture based on various aspects of the incoming request."

"There are three intrinsic implementations of the IRequestCultureProvider interface:"

Note that the request culture providers don't actually set the culture on the thread themselves. That's done by the middleware.